src/llama.cpp

```diff
@@ -11736,6 +11737,7 @@ struct llm_build_context {
 
                 // ref: https://github.com/google/gemma_pytorch/commit/03e657582d17cb5a8617ebf333c1c16f3694670e
                 switch (model.type) {
+                    case e_model::MODEL_2B: Qcur = ggml_scale(ctx0, Qcur, 1.0f / sqrtf(float(n_embd_head_k))); break;
```

Comment:
Reuse the 9B implementation:
```suggestion
                    case e_model::MODEL_2B:
```